Jenkinsfile

```diff
@@ -1,11 +1,31 @@
 node {
-	stage 'checkout'
+  try {
+    stage 'checkout'
 
-	checkout scm
+    checkout scm
 
-	stage 'cibuild'
-	sh "scripts/cibuild"
-	
-	stage 'cipublish'
-	sh "sleep 3;"
-}
\ No newline at end of file
+    stage 'cibuild'
+
+    wrap([$class: 'AnsiColorBuildWrapper']) {
+      sh 'scripts/cibuild'
+    }
+
+    stage 'cipublish'
+
+    env.AWS_DEFAULT_REGION = 'us-east-1'
```

Comment:
Should `AWS_DEFAULT_REGION` be scoped to the `if (env.BRANCH_NAME == 'develop')` stanza? It doesn't look like it's being used outside of there.